### ISSUE TITLE
Why related pages cannot be fetched via the Pages page type field?

### ISSUE THREAD

**Issue by JanLenoch** [Kentico/xperience-training-13#75]
### Brief bug description

For an unknown reason, fetching of related pages throws NRE exceptions.

### Repro steps

1. Run the home page
2. Observe the MapDtoProperties method in the DoctorRepository and HomePageRepository classes.

### Expected behavior

What is the correct behavior?

### Test environment

 - Platform/OS: [e.g. .NET Core 3.1, iOS, Windows]
 - Browser [e.g. Chrome, Safari, Edge Chromium, Edge]
 - Version [e.g. 81]

### Additional context

Add any other context about the problem here. If you found an issue where a similar bug has been reported, add a link here.

### Screenshots

Add links to screenshots, if you deem it appropriate.

### Issue expands upon

This issue expands upon another issue(s): #`issue number`

### Blockers

This issue is blocked by the following issue(s): #`issue number`
